client/components/App.jsx

```diff
@@ -0,0 +1,73 @@
+// functional stateful class
+import React from 'react';
+import $ from  'jquery'
+import Carousel from './Carousel.jsx'
+
+class App extends React.Component {
+  constructor (props){
+    super(props)
+    this.state = {
+    data: [],
+    frontImages: [],
+    mainImg: [],
+    page: 1,
+    view: false
+    }
+    this.switchToCarousel =this.switchToCarousel.bind(this);
+  }
+  // component mount does ajax request
+  componentDidMount() {
+  $.ajax({
+    method: 'POST',
```

Comment:
I'm note sure, but I think that in here you're trying to do a GET request, to fetch data from the server.
I think you did correctly to call it inside the componentDidMount, but I suggest to you look if the POST method is what you want to use here.

Reply:
Im sending my item page number instead of getting all the data